.github/workflows/build-jsp.yml

```diff
@@ -17,6 +17,9 @@ jobs:
       with:
         repository: unicode-org/cldr
         path: cldr
+    - name: Backup Unicodetools and CLDR for jsps  # this is needed only for the Docker build
```

Comment:
dumb question: Where does it get backed up to, and how does another step find it?

Reply:
not a dumb question.  It gets copied to `UnicodeJsps/target/cldr-unicodetools.tgz` as a temporary file, and is then picked up in the Dockerfile. It's a copy of the repositories not including any build artifacts or git metadata, to keep the docker image smaller.